python/README.md

```diff
@@ -22,6 +22,9 @@
    pip install -r requirements.txt
    ```
 
+   If you need to use Azure or GCP KMS, follow our instructions on
+   [how to build libmongocrypt from source](https://github.com/mongodb/libmongocrypt/tree/master/bindings/python#installing-from-source).
+
```

Comment:
```suggestion
  A required dependency has been omitted from `requirements.txt`: `pymongocrypt`.
   If you need to use Azure or GCP KMS, follow our instructions on
   [how to build libmongocrypt from source](https://github.com/mongodb/libmongocrypt/tree/master/bindings/python#installing-from-source).
  Otherwise, you can install it via `pip install pymongocrypt==1.0.1`
```